packages/website/docusaurus.config.js

```diff
@@ -67,8 +67,9 @@ const config = {
         // hideOnScroll: true,
         logo: {
           alt: 'TypeScript ESLint',
+          height: '2rem',
           src: 'img/logo.svg',
-          // srcDark: 'img/logo-dark.svg',
+          width: '2rem',
```

Comment:
Images should have explicit height and width before they load. That way the browser knows how to resize them immediately.

Doesn't seem to be a real issue for us as the header is a fixed size but might as well improve for SEO 🤷